package.json

```diff
@@ -34,7 +34,7 @@
         "datatables.net-select-bs5": "1.4.0",
         "dropzone": "4.3.0",
         "jqtree": "1.7.0",
-        "jquery": "^3.6.4",
+        "jquery": "^3.7.0",
```

Comment:
I tried that out a while ago, but this breaks the tests. Didn't look into details yet.

Reply:
jQuery is not the problem. See comment below.